packages/instrumentation-mysql2/README.md

```diff
@@ -54,19 +54,28 @@ You can set the following instrumentation options:
 
 ## Semantic Conventions
 
-This package uses `@opentelemetry/semantic-conventions` version `1.22+`, which implements Semantic Convention [Version 1.7.0](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.7.0/semantic_conventions/README.md)
+This instrumentation implements Semantic Conventions (semconv) v1.7.0. Since then, networking (in semconv v1.23.1) and database (in semconv v1.33.0) semantic conventions were stabilized. As of `@opentelemetry/instrumentation-mysql2@0.53.0` support has been added for migrating to the stable semantic conventions using the `OTEL_SEMCONV_STABILITY_OPT_IN` environment variable as follows:
+
+1. Upgrade to the latest version of this instrumentation package.
+2. Set `OTEL_SEMCONV_STABILITY_OPT_IN=http/dup,database/dup` to emit both old and stable semantic conventions. (The `http` token is used to control the `net.*` attributes, the `database` token to control to `db.*` attributes.)
+3. Modify alerts, dashboards, metrics, and other processes in your Observability system to use the stable semantic conventions.
+4. Set `OTEL_SEMCONV_STABILITY_OPT_IN=http,database` to emit only the stable semantic conventions.
+
```

Comment:
```suggestion

By default, if `OTEL_SEMCONV_STABILITY_OPT_IN` includes neither of the above tokens, the old v1.7.0 semconv is used.
The intent is to provide an approximate 6 month time window for users of this instrumentation to migrate to the new database and networking semconv, after which a new minor version will use the new semconv by default and drop support for the old semconv.
```

I added this to the PG one, to make things clear there is a timeframe for the update and what to expect if you don't set the env variable

Reply:
Actually on a re-read of the migration guide https://opentelemetry.io/docs/specs/semconv/non-normative/db-migration/ (same for the http guide):

> May drop the environment variable in their next major version and emit only the stable database conventions.

I think we should consider only dropping these in OTel JS SDK v3... which we'll do to drop Node.js 18 support (and likely Node.js 20 support) about Jun 2026.  I guess, given all these instrumentations are 0.x it *will* be a "minor" version, but *breaking* change, so [shrug/].